package-url.md

```diff
@@ -0,0 +1,48 @@
+# purl format for Hex
+
+## Introduction
+
+[purl or Package URL](https://github.com/package-url/purl-spec) is a
+specification for identifying and locating software packages. It may be used
+to uniquely identify a package or package version in a software
+bill-of-materials (SBOM) or vulnerability report, or to locate the package in a
+repository, e.g. to check for updates.
+
+In particular, the [CycloneDX](https://cyclonedx.org) SBOM specification
+uses purl to identify components.
+
+This document formalizes the purl format for packages hosted on hex.pm or
+compatible repositories. Like Hex itself, it applies equally to packages
+written in/for Erlang, Elixir or any other BEAM language.
+
+## Definition
+
+* The 'type' is "hex"
+* The default repository is https://hex.pm
+* The 'namespace' is optional; it may be used to specify the organization for
+  private packages on hex.pm; it is not case sensitive and must be lowercased.
```

Comment:
Is this only for packages on hex.pm? Or should it support repositories on other hosts?

Reply:
Nevermind, I see`repository_url` below.